torchinductor/codecache.py

```diff
@@ -68,6 +68,18 @@ def cpp_compiler_search(search):
             continue
     raise exc.InvalidCxxCompiler()
 
+@functools.lru_cache(1)
```

Comment:
```suggestion
@functools.lru_cache(128)
```
Larger cache size limit